prow/config.yaml

```diff
@@ -14752,6 +14752,21 @@ periodics:
     - name: docker-graph
       emptyDir: {}
 
+- interval: 6h
+  name: ci-kubernetes-multicluster-ingress-test
+  agent: kubernetes
+  labels:
+    preset-service-account: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+      args:
+      - --repo=github.com/GoogleCloudPlatform/k8s-multicluster-ingress=master
+      - --root=/go/src
+      - --timeout=50
+      - --service-account=/etc/service-account/service-account.json
+      - --upload=gs://kubernetes-jenkins/logs
```

Comment:
does this job need `k8s-multicluster-ingress-coveralls-token`?

Reply:
Right added. I am not sure how is the secret created and if I need to do anything to ensure that secret?

Reply:
secrets are manually added to the prow cluster, if you're already using this secret it should be fine. I vaguely remember adding one for this.

Reply:
Yes our pull job uses the same secret and is running fine so I assume it exists.